<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChucNang</class>
 <widget class="QWidget" name="ChucNang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ChucNang{
border-image: url(:/newPrefix/Images/chucnang.png);
}
QPushButton{
background-color: rgba(0, 0, 0, 0);
border-radius: 30px;
}
QPushButton:hover{
background-color: rgba(0, 0, 0, 20);
}</string>
  </property>
  <widget class="QPushButton" name="btnThongTinNguoiDung">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>101</y>
     <width>161</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLenKeHoach">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>161</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLichSu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>161</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnKhamPhaMon">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>161</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
